release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -24,7 +24,14 @@ submodule openconfig-bgp-common-multiprotocol {
     for multiple protocols in BGP. The groupings are common across
     multiple contexts.";
 
-  oc-ext:openconfig-version "9.7.1";
+  oc-ext:openconfig-version "9.7.2";
+
+  revision "2024-07-25" {
+    description
+      "Clarification to usage of container
+      `.../use-multiple-paths/ebgp/link-bandwidth-ext-community/` ";
```

Comment:
```suggestion
      link-bandwidth-ext-community.";
```
(ditto for the other files)

Reply:
I'm not sure if "link-bandwidth-ext-community" string do not appear in other places of hierarchy. This way I give a context.